packages/visual-editing/src/controller.ts

```diff
@@ -312,6 +298,7 @@ export function createOverlayController({
       id,
       rect: getRect(element),
       sanity,
+      dragDisabled: element.getAttribute('data-sanity-disable-drag') === 'false',
```

Comment:
As I understand it `getAttribute()` will return `'true'` (as a string) if the data attribute value is empty, and `null` if the attribute is missing?

So this would evaluate to true (i.e. drag is disabled) when rendering the following?:
`data-sanity-disable-drag="false"`

Reply:
Ah, good catch. We need to check for `true` rather than `false`! 🕵️